week-2/Homework/extra/4-radio-stations.js

```diff
@@ -35,11 +66,14 @@
  * - "No station found at ${frequency}, moving on"
  */
 // `searchRadioWaves` goes here
+function searchRadioWaves() {}
```

Comment:
Hey @babakjahangiri , in future, you could reach out to a volunteer for clarification (apologies if you've already done that and I've missed it), but thanks for submitting your homework instead of leaving it till late! :) I also appreciate there's no test to help you identify what the `searchRadioWaves` function should look like. One approach could be to randomly select an element returned from the `getAllFrequencies` function and seeing if it is a radio station using the output from the `getStations` function. If the elements match, you can log to console `station found`, otherwise `no station found` (like when you press a button in a car's radio and sometimes it returns a radio station and other times it doesn't).